sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -1271,6 +1271,23 @@ class SubquerySuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("Cannot remove sort for floating-point order-sensitive aggregates from subquery") {
+    Seq("float", "double").foreach { typeName =>
+      Seq("SUM", "AVG", "KURTOSIS", "SKEWNESS", "STDDEV_POP", "STDDEV_SAMP",
+          "VAR_POP", "VAR_SAMP").foreach { aggName =>
+        val query1 =
```

Comment:
nit. `query1` -> `query` since we remove the other test query.

Reply:
@maropu , you can ignore this renaming comment.

Reply:
oh, right.